tests/test_enum.py

```diff
@@ -247,7 +247,7 @@ class IceCreamFlavour(Enum):
     @strawberry.input
     class Input:
         flavour: IceCreamFlavour
-        optionalFlavour: typing.Optional[IceCreamFlavour]
```

Comment:
uhm, why this change?

Reply:
The default allows it to be instantiated with a missing arg.  (and to test that being explicitly `Optional` still works)

Reply:
I'm wondering if it is ok to not set none as default automatically, guess explicit is better 🤔